src/views/virtualmachines/details/tabs/diagnostic/utils/utils.ts

```diff
@@ -93,3 +93,16 @@ export const buildDataVolumeResources = (vm: V1VirtualMachine) =>
       },
     ]) || [],
   );
+
+export const createURLDiagnostic = (str: string, append: string): string => {
+  const urlSpitted = str.split('/');
+  if (
+    urlSpitted[urlSpitted.length - 1] === 'logs' ||
+    urlSpitted[urlSpitted.length - 1] === 'tables'
```

Comment:
can you use the consts here too? VirtualMachineDetailsTab.Tables and VirtualMachineDetailsTab.Logs

Reply:
sure, done